test/specs/rules/require-sorted-object-keys.js

```diff
@@ -0,0 +1,49 @@
+var Checker = require('../../../lib/checker');
+var assert = require('assert');
+
+describe('rules/require-sorted-object-keys', function() {
+    var checker;
+
+    beforeEach(function() {
+        checker = new Checker();
+        checker.registerDefaultRules();
+    });
+
+    it('should alias option value true to asc', function() {
+        checker.configure({ requireSortedObjectKeys: true });
+
+        assert(checker.getConfiguration().getConfiguredRule('requireSortedObjectKeys').sort === 'asc');
```

Comment:
Right, I understand that this layer (parsed configuration rules) may be too *far away* from this test. I wanted to avoid code repetition, but if you feel it's more inline with the general test model, it's fine by me.